system_strings/sysmes_text.en

```diff
@@ -467,11 +467,11 @@ Slumber
 Ordinary Homeroom
 I eat in the classroom.
 Should I go to the tea room...?
-...No, I created this reality.
-Unstopping Rain
-Guidance
-Confession
-Listening to the sound of rain
+……いや、俺の起こした現実だ。
```

Comment:
Reverts to JP here intentional?

Reply:
yeah it's intentional. those are from ciel route